tests/lib.rs

```diff
@@ -4,6 +4,11 @@ extern crate core;
 
 extern crate safe_transmute;
 
+#[cfg(feature = "std")]
```

Comment:
My only question for this commit (df5346627b3ee712e0fbbfff8991b32d63ad228b) in its entirety is: why?

Reply:
Because `include!`s are a mess. If I don't centralise these `use`s to a single place, there will be warnings or errors. Perhaps we should get rid of them entirely and make it an actual module?

Reply:
Can't make it an "actual module" because we're including it in doctests. argh, schite.

Allllllllllllllllllllllllllllllllllllllllllllllllllrighty then, as much as I'm pained to do this, the only reasonable-ish solution *is* to move the `use`s into the funxion bodies, innit :v

Reply:
That was a bit unclear. Do you want me to move the `use`s then?

Reply:
Probably, if there's no better way, aye

Reply:
It is done in 1d79fa6.